File: 67-add-binary/add-binary.js
/**
 * @param {string} a
 * @param {string} b
 * @return {string}
 */
var addBinary = function(a, b) {
    let br1=BigInt("0b"+a);
    let br2=BigInt("0b"+b);
    let res=br1+br2;
    return res.toString(2);
};

File: 67-add-binary/add-binary.py
class Solution(object):
    def addBinary(self, a, b):
        """
        :type a: str
        :type b: str
        :rtype: str
        """
        br1=int(a,2)
        br2=int(b,2)
        res=br1+br2
        return bin(res)[2:]